core/trino-main/src/main/java/io/trino/connector/system/jdbc/ColumnJdbcTable.java

```diff
@@ -431,60 +431,60 @@ static Integer columnSize(Type type)
         if (type.equals(TINYINT)) {
             return 3;   // 2**7-1
         }
-        if (type instanceof DecimalType) {
```

Comment:
It is a bit controversial. I generally like this check - but it requires what may not be seen as universally better. It forces you to pick name for a variable to avoid cast.

Reply:
yeah, and why this is an issue?

Reply:
Sometimes it is tricky to come up with reasonable name given what is alrady in the namespace. Probably not a big deal.